### DISCUSSION TITLE
colspan with v7

### DISCUSSION THREAD

**Discussion by milenaNeumann** [TanStack/table#2703] [General]
Hi there,

I have read [here](https://github.com/tannerlinsley/react-table/issues/1342) that colspan will be (or is already?) featured in v7.
I cannot seem to find documentation or a working example on the topic.
What is the status of this information? 

Best wishes ;)
Milena

**Comment by ruinobreferreira** [comment]
+1

**Reply by Josh68** [reply]
Hmm, I'm wondering if anyone monitors these discussions. 👍

**Reply by Josh68** [reply]
I ended up doing this, which seems a bit hacky (and could likely be written better)

```js
  // Empty row
  // data config
  [
    {
      accessorName: null
    }
  ]
```

"Template"
```js
row.cells.map((cell, index, _array) => {
                  let i = index;
                  const colSpanValue = _array.reduce(acc => {
                    const nextCell = _array[i + 1];
                    if (nextCell && !nextCell?.value) {
                      acc++;
                      i++;
                    }
                    return acc;
                  }, 1);
                  if (!cell?.value) {
                    return null;
                  }

  ...

  conditionalProps.colSpan=colSpanValue
  ...
  
  <TableCell {...conditionalProps}...>
    {cell.render("Cell")}
  </TableCell>  
```

**Comment by SaraRavasio** [comment]
Are there any updates? Can we pass a `colSpan` property? seems like a very important and "basic" feature to have
